src/Compressors/Gaussian.jl

```diff
@@ -0,0 +1,154 @@
+"""
+    Gaussian <: Compressor
+
+An implementation of the Gaussian compression method. This method forms a Gaussian sketch matrix
+with number of rows or number of columns setting as the compression dimension.
```

Comment:
A simple description may be better. E.g., `A specification for a Gaussian compressor.`